### DISCUSSION TITLE
[T2] Duda parte 2 formato paquetes

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#134] [Tarea 2] [answered]
Tengo una duda respecto a los paquetes, ya que se menciona de que siempre serán de tamaño 3x3, pero no se menciona si tendrán otras características compartidas, como que la esquina inferior izquierda esté siempre ocupada o que serán formas sin saltos. Por ejemplo serían estas cargas apropiadas?

A
1 0 1
0 1 0
1 0 0

B
1 1 1
1 0 1
0 0 1

**Comment by sebacarrascop** [comment, marked as answer]
hola, dado que no está en las consideraciones, no puedes hacer ninguno de esos supuestos
